doc/modules/model_evaluation.rst

```diff
@@ -584,13 +584,15 @@ and inferred labels::
    >>> y_pred = [0, 0, 2, 1, 0]
    >>> target_names = ['class 0', 'class 1', 'class 2']
    >>> print(classification_report(y_true, y_pred, target_names=target_names))
-                precision    recall  f1-score   support
+                 precision    recall  f1-score   support
```

Comment:
Is the doctest failing before?.

Reply:
We should probably not change anything if this is not the reason.

Reply:
The existing code adjusts the column width to fit the maximum column. Since we now have a longer "weighted avg" column name, everything needs indentation.